src/Integration/Resources/Strings.resx

```diff
@@ -433,4 +433,8 @@ Localization note: '[Don't warn me again]()' is a special markup used to transla
     <value>Not all NuGet packages were installed. Please see output above for more information.</value>
     <comment>Output message indicating not all NuGet packages were installed during the binding workflow</comment>
   </data>
+  <data name="SolutionSaveCancelledBindAborted" xml:space="preserve">
+    <value>Solution save was cancelled. Bind operation was aborted.</value>
```

Comment:
I don't think it's too verbose for the output window, although it might be a bit long for a dialogue box. 

How about:
"Cannot bind if there are unsaved changes. Operation cancelled." or
"Cannot bind if there are unsaved changes. Please save changes and try again."
Will the same message appear when sync-ing (in which case it should perhaps say "cannot bind or sync...")?